src/EditorFeatures/TestUtilities/Workspaces/TestWorkspace_XmlConsumption.cs

```diff
@@ -346,7 +346,9 @@ public static TestWorkspace Create(string xmlDefinition, bool openDocuments = fa
 
                 var name = GetFileName(workspace, sourceGeneratedDocumentElement, ref documentId);
 
-                var markupCode = sourceGeneratedDocumentElement.NormalizedValue();
+                var markupCode = (bool?)sourceGeneratedDocumentElement.Attribute(NormalizeAttributeName) is false
```

Comment:
love this less :)

Reply:
By default, it still normalizes in cases where the input is the workspace XML syntax instead of string literals. A follow-up could remove normalization across the board.